contracts/BdStable/Pools/BdPoolLibrary.sol

```diff
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.6.12;
-pragma experimental ABIEncoderV2;
+pragma solidity 0.8.13;
 
-import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
```

Comment:
I agree we can remove it, but for now didn't want to start changing even more in the contracts.. Too much already.
It doesn't harm as it's just a wrapper around the existing regular operators.. SO this could be done as a tech debt somewhere else.

WDYT?

Reply:
we are already going over the app and testing everything... let's do it now so we won't need to retest everything again on another phase

Reply:
I would recommend doing it in a dedicated PR once this will be merged. Too many changes already.

Reply:
You know what, let me have a look again how big it will be

Reply:
By the way, there is not harm to keep it there to be honest.
If you'll look at its implantation in this version, it does nothing. Just wrapped the native operators.
Sub for example is just:
```solidity
Return a - b
```

Reply:
Yes, I just didn't understand why to change it specifically in BDStable and not in other files

Reply:
We'll actually I did t change it there SafeMath wasn't used there at all.

What happened here it that BDStable uses the BDStableLibrary which uses SafeMath, so by doing it the code on BDStable was able to "use SafeMath".
That was wrong to begin with and shouldn't be done. While the 0.6 compiler allowed it, the 0.8 did not.
So the code ase did not compile.

I could have fixed it by using SafeMath in the BDstable contract, but thought this is better.

Reply:
OK I've removed all the places we were using SafeMath.
Please take the time to carefully review my changes as some places could be a bit confusing.